tests/integration/integration_test_case.py

```diff
@@ -193,6 +193,15 @@ def sign_out(self):
         selected = self.getHtmlSoup().find("a", {"name": "btn-save-sign-out"})
         return self.get(selected["href"])
 
+    def exit(self):
+        """
+        GETs the sign-out url from the exit button. Does not follow the external
+        redirect.
+        """
+        url = self.getHtmlSoup().find("a", {"name": "btn-exit"})["href"]
+        environ, response = self._client.get(url, as_tuple=True, follow_redirects=False)
+        self._cache_response(environ, response)
```

Comment:
We should make `follow_redirects` a param of `get` (defaulted to `True`). This could then be:
```suggestion
        self.client.get(url, follow_redirects=False)
```

Reply:
Yep, that's a cleaner implementation. I'll update.